.travis.yml

```diff
@@ -5,7 +5,7 @@ matrix:
 
 # command to install dependencies
 install:
-  - pip install pytest Pillow pycodestyle
+  - pip install -r requirements.txt
```

Comment:
these are missing pytest and pycodestyle now

Reply:
Hrm, seems to be included in the python image then. The docs even reference pytest https://docs.travis-ci.com/user/languages/python/
So we should be good